CMakeLists.txt.user

```diff
@@ -0,0 +1,645 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should indeed be added in .gitignore.

Reply:
You should also delete the file itself, I believe this will still get added to the repo because you've added the gitignore changes afterwards. @nuttyartist